srcJava/com/rti/perftest/ddsimpl/PerftestTransport.java

```diff
@@ -789,11 +817,69 @@ private void configureShmemTransport(DomainParticipantQos qos) {
 
     public boolean configureTransport(DomainParticipantQos qos) {
 
-        switch (transportConfig.kind) {
+        /*
+        * If transportConfig.kind is not set, then we want to use the value
+        * provided by the Participant Qos, so first we read it from there and
+        * update the value of transportConfig.kind with whatever was already set.
+        */
+        if (transportConfig.kind == Transport.TRANSPORT_NOT_SET) {
+
+            int mask = qos.transport_builtin.mask;
+            switch (mask) {
+                case TransportBuiltinKind.UDPv4:
+                    transportConfig = new TransportConfig(
+                            Transport.TRANSPORT_UDPv4,
+                            "UDPv4",
+                            "dds.transport.UDPv4.builtin");
+                    break;
+                case TransportBuiltinKind.UDPv6:
+                    transportConfig = new TransportConfig(
+                        Transport.TRANSPORT_UDPv6,
```

Comment:
add 4 spaces here and in the following functions

Reply:
Done.